source/Module/STIG/Functions.Checklist.ps1

```diff
@@ -166,219 +214,221 @@ function New-StigCheckList
+                        {
+                            $status = $statusMap['Open']
+                            $comments = "Configuration attempted by PowerStig MOF via $setting, but not currently set."
+                            $findingDetails = Get-FindingDetails -Setting $setting
+                        }
+                        else
+                        {
+                            $status = $statusMap['Open']
+                        }
                     }
                     else
                     {
-                        $status = $statusMap['Open']
-                    }
+                        $status = $statusMap['NotReviewed']
+                    }    
                 }
-                else
-                {
-                    $status = $statusMap['NotReviewed']
-                }    
             }
-        }
 
-        # Test to see if this rule is managed as a duplicate
-        $convertedRule = $processed.SelectSingleNode("//Rule[@id='$vid']")
+            # Test to see if this rule is managed as a duplicate
+            $convertedRule = $processed.SelectSingleNode("//Rule[@id='$vid']")
 
-        if ($convertedRule.DuplicateOf)
-        {
-            # How is the duplicate rule handled? If it is handled, then this duplicate is also covered
-            if ($PSCmdlet.ParameterSetName -eq 'mof')
+            if ($convertedRule.DuplicateOf)
             {
-                $originalSetting = Get-SettingsFromMof -ReferenceConfiguration $referenceConfiguration -Id $convertedRule.DuplicateOf
-
-                if ($originalSetting)
+                # How is the duplicate rule handled? If it is handled, then this duplicate should have the same status
+                if ($PSCmdlet.ParameterSetName -eq 'mof')
                 {
-                    $status = $statusMap['NotAFinding']
-                    $findingDetails = 'See ' + $convertedRule.DuplicateOf + ' for Finding Details.'
-                    $comments = 'Managed via PowerStigDsc - this rule is a duplicate of ' + $convertedRule.DuplicateOf
-                }
-            }
-            elseif ($PSCmdlet.ParameterSetName -eq 'result')
-            {
-                $originalSetting = Get-SettingsFromResult -DscResult $dscResult -id $convertedRule.DuplicateOf
+                    $originalSetting = Get-SettingsFromMof -MofFile $MofFile -Id $convertedRule.DuplicateOf
 
-                if ($originalSetting.InDesiredState -eq 'True')
-                {
-                    $status = $statusMap['NotAFinding']
-                    $findingDetails = 'See ' + $convertedRule.DuplicateOf + ' for Finding Details.'
-                    $comments = 'Managed via PowerStigDsc - this rule is a duplicate of ' + $convertedRule.DuplicateOf
+                    if ($originalSetting)
+                    {
+                        $status = $statusMap['NotAFinding']
+                        $findingDetails = 'See ' + $convertedRule.DuplicateOf + ' for Finding Details.'
```

Comment:
Can you use the format operator instead of string concatenation ?  
$findingDetails = 'See {0} for Finding Details.' -f $convertedRule.DuplicateOf

Reply:
Done. What difference does it make? Don't they do exactly the same thing?

Reply:
They guys have been wanting new code to use string.format..
https://github.com/microsoft/PowerStig/pull/648 - 
![image](https://user-images.githubusercontent.com/44269812/86250884-28ebbc80-bb7f-11ea-88de-030e4390e52c.png).  
String Format.. Null arguments are handled a bit different
https://docs.microsoft.com/en-us/dotnet/api/system.string.format?view=netcore-3.1